README.md

```diff
@@ -166,6 +166,13 @@ The `--prebuiltDirectory` flag is useful for working with Webpack for example. I
 ## Handling `npm link` and Dependencies With Local Paths
 Perhaps the easiest way to handle these cases is to bundle the code using Webpack and use the `--prebuiltDirectory` flag to package the output for deployment.
 
+## ScheduleEvents
+#### Optional Parameter
+When using the eventSourceFile flag (-S or --eventSourceFile) to set a ScheduleEvent trigger, you can pass an optional _ScheduleDescription_ key into the ScheduleEvent object with a custom description for the CloudWatch event rule you are defining. By default, node-lambda generates a _ScheduleDescription_ for you based on the ScheduleName and ScheduleExpression of the rule.
+
+#### Note on ScheduleState for ScheduleEvents
```

Comment:
Maybe not exactly how we like it, but long as it's been documented I think we're in a better state than before already. Thanks for that!